ruby-challenges.rb

```diff
@@ -3,30 +3,116 @@
 
 # --------------------1) Create a method that takes in an array of words and a single letter and returns an array of all the words containing that particular letter. Use the test variables provided.
 
+# A method that takes in an array of words and a single letter
+def contains_letter(arrayWords, letter)
```

Comment:
It can be easy to overthink Ruby code.  While this will work, the main logic needed for this method exists on lines 13, and 14.